js/main.js

```diff
@@ -143,4 +173,12 @@ $submitForm.addEventListener('submit', submitForm);
 
 $ul.addEventListener('click', editEntry);
 
+$deleteEntry.addEventListener('click', modalOn);
+
+$cancelBtn.addEventListener('click', modalOff);
+
+$confirmDelete.addEventListener('click', deleteEntry);
+
 switchView(data.view);
+
+data.nextEntryId = data.entries.length + 1;
```

Comment:
You should remove this, it can cause problems with multiple items with the same id.